.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -12,7 +12,7 @@ else
         '--detach', 'mcr.microsoft.com/mssql/server:2019-latest'
     )
 
-    docker pull mcr.microsoft.com/mssql/server &&
-    docker run @dockerRunSwitches &&
+    docker pull mcr.microsoft.com/mssql/server
+    docker run @dockerRunSwitches
```

Comment:
Why did you remove these? Aren't these working across lines?

Reply:
Also incompatible with Windows PowerShell (it looked like a mistake).